### PR TITLE
[MAINT] Update Decommissioning Email Templates

### PR DESCRIPTION
**Pull request by jacob-ward**
### Description:

Updates to the decommissioning email templates to reflect the 'hard' end-of-life date for the C flavor hardware.

### Special Notes:

---

### Submitter:

Have you (where applicable):

* [ ] Added unit tests?
* [ ] Checked the latest commit runs on Dev?
* [ ] Updated the example config file(s) and README?

---

### Reviewer

Does this PR:

* [ ] Place non-StackStorm code into the `lib` directory?
* [ ] Have unit tests for the action/sensor and `lib` layers?
* [ ] Have clear and obvious action parameter names and descriptions?

### CODE REVIEW

File: email_templates/plaintext/decom_email.j2 [hunk 1]
```diff
@@ -24,9 +25,9 @@ Where possible we will not decommission hardware that is still hosting User owne
 if the<br>Hypervisor hosting your VM fails we have NO spares and will NOT attempt anything beyond basic repair work,
 <br>in this case you will suffer the total loss of your VM.
 
-Failure to comply with this notice will eventually lead to Hardware being turned off and disposed of, whether it<br>
-is still hosting VMs or not, eventually we will have no choice. Routine reminders will follow and increase in
-frequency over the coming weeks.
+Failure to comply with this notice by mid day (12:00PM) on Wednesday 28th February 2024 will lead to Hardware being 
+turned off and disposed of, whether it<br> is still hosting VMs or not, we have no choice at that point. Routine
```
Comment (meoflynn) [inline email_templates/plaintext/decom_email.j2:29]: There's a `<br>` which I think is from the HTML version of the message

**Review by meoflynn — CHANGES_REQUESTED** [review]

File: email_templates/html/decom_email.html.j2 [hunk 2]
```diff
@@ -25,9 +26,9 @@ Where possible we will not decommission hardware that is still hosting User owne
 if the<br>Hypervisor hosting your VM fails we have NO spares and will NOT attempt anything beyond basic repair work,
 <br>in this case you will suffer the total loss of your VM.<br><br>
 
-Failure to comply with this notice will eventually lead to Hardware being turned off and disposed of, whether it<br>
-is still hosting VMs or not, eventually we will have no choice. Routine reminders will follow and increase in
-frequency over the coming weeks.<br><br>
+Failure to comply with this notice by mid day (12:00PM) on Wednesday 28th February 2024 will lead to Hardware being
```
Comment (anish-mudaraddi) [inline email_templates/html/decom_email.html.j2:29]: One for the future but we should add a variable like `{{ Time }}` and `{{ Date }}` to make this template more generic

**Review by anish-mudaraddi — APPROVED** [review]
LGTM

File: email_templates/plaintext/decom_email.j2 (hunk 1 above)
Comment (jacob-ward) [reply email_templates/plaintext/decom_email.j2]: Thanks! Fixed in latest commit

File: email_templates/html/decom_email.html.j2 (hunk 2 above)
Comment (jacob-ward) [reply email_templates/html/decom_email.html.j2:29]: Yeh, was thinking something similar given that we'll be doing this for at least 1 generation per year but yes - change for down the road

**Review by meoflynn — APPROVED** [review]

**Review by anish-mudaraddi — APPROVED** [review]

**Pull request merged**